engine/examples/mongodb.md

```diff
@@ -69,53 +72,60 @@ Continuing, we will declare the `MAINTAINER` of the `Dockerfile`:
 We will begin with importing the MongoDB public GPG key. We will also create
 a MongoDB repository file for the package manager.
 
-    # Installation:
-    # Import MongoDB public GPG key AND create a MongoDB list file
-    RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
-    RUN echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-3.0.list
+```dockerfile
+# Installation:
+# Import MongoDB public GPG key AND create a MongoDB list file
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+RUN echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-3.0.list
+```
 
 After this initial preparation we can update our packages and install MongoDB.
 
-    # Update apt-get sources AND install MongoDB
-    RUN apt-get update && apt-get install -y mongodb-org
+```dockerfile
+# Update apt-get sources AND install MongoDB
+RUN apt-get update && apt-get install -y mongodb-org
+```
 
-> **Tip:** You can install a specific version of MongoDB by using a list
-> of required packages with versions, e.g.:
->
->     RUN apt-get update && apt-get install -y mongodb-org=3.0.1 mongodb-org-server=3.0.1 mongodb-org-shell=3.0.1 mongodb-org-mongos=3.0.1 mongodb-org-tools=3.0.1
+> **Tip:** You can install a specific version of MongoDB by using a list of required packages with versions, e.g.:
+```dockerfile
+RUN apt-get update && apt-get install -y mongodb-org=3.0.1 mongodb-org-server=3.0.1 mongodb-org-shell=3.0.1 mongodb-org-mongos=3.0.1 mongodb-org-tools=3.0.1
+```
 
 MongoDB requires a data directory. Let's create it as the final step of our
 installation instructions.
 
-    # Create the MongoDB data directory
-    RUN mkdir -p /data/db
+```dockerfile
+# Create the MongoDB data directory
+RUN mkdir -p /data/db
+```
 
 Lastly we set the `ENTRYPOINT` which will tell Docker to run `mongod` inside
 the containers launched from our MongoDB image. And for ports, we will use
 the `EXPOSE` instruction.
 
-    # Expose port 27017 from the container to the host
-    EXPOSE 27017
+```dockerfile
+# Expose port 27017 from the container to the host
```

Comment:
Hm, something we should fix "expose" doesn't actually _expose_, it only adds metadata to the dockerfile to describe that the image "exposes" a certain port (it's used for the `-P` (publish all) option)